pkg/image/nbdkit.go

```diff
@@ -126,7 +126,8 @@ func NewNbdkitCurl(nbdkitPidFile, user, password, certDir, socket string, extraH
 	return n, nil
 }
 
-// Keep these in a struct to keep NewNbdkitVddk from going over the argument limit
+// NbdKitVddkPluginArgs keeps these arguments in a struct to keep NewNbdkitVddk
+// from going over the linter limit.
```

Comment:
You can drop this commit since we don't use this linter in main anymore

Reply:
Okay, gone.